### DISCUSSION TITLE
Trviy operator doesn't use ICSP/IDMS in OpenShift

### DISCUSSION THREAD

**Discussion by AndreasDavour** [aquasecurity/trivy-operator#2246] [Help & Support]
Hi

I have deployed the Trivy Operator v.0.22 in OpenShift, but our environment i air-gapped so it poses some specific challenges. We have ICSP/IDMS configured, so OpenShift uses its own registry mirroring for images. But, when Trivy starts scanning, it tries to download images as they are named in the deployments in memory, not using the ICSP/IDMS config to go through out proxied registry.

Have I missed some config, or is Trivy not able to use the image digest mirror set to find out how to pull and scan the running images?

**Comment by mushroom-hat** [comment]
+1

**Comment by AndreasDavour** [comment]
I decided to look for other solutions.

**Comment by alifib** [comment]
could you publish a guide that shows how to configure the operator with RedHat OpenShift . especially considering an air-gapped environment where ICSP and IDMS is configured?
